index.html

```diff
@@ -128,74 +149,190 @@ <h2 class="card-head">Multi-Post <br>Stories</h2>
+        <div class="about-first-part">
+            <h2 class="about-title">About <br>Myself</h2>
+        
+            <div class="body-container">
+                <p>
+                    I'm a growing software developer! I can help you build web app or website. Have a look 
+                    through some of my works. If you like what you see and have a project you need coded, don't hesitate to contact me.
+                </p>
+            </div>
+
+            <h3 class="contact">
+                LET'S CONNECT
+            </h3>
         
+            <ul class="contact-list">
+                <li><a class="contact-link" href="https://twitter.com/msd_811" rel="noopener noreferrer" target="_blank"><img src="images/twitter.svg" alt="Twitter"></a></li>
+                <li><a class="contact-link" href="https://www.linkedin.com/in/msd811/" rel="noopener noreferrer" target="_blank"><img src="images/linkedin-icon.svg" alt="LinkedIn"></a></li>
+                <li><a class="contact-link" href="https://www.linkedin.com/in/msd811/" rel="noopener noreferrer" target="_blank"><img src="images/m.svg" alt="M"></a></li>
+                <li><a class="contact-link" href="https://github.com/Masuduzzaman811" rel="noopener noreferrer" target="_blank"><img src="images/github.svg" alt="GitHub"></a></li>
+                <li><a class="contact-link" href="https://www.linkedin.com/in/msd811/" rel="noopener noreferrer" target="_blank"><img src="images/hand-logo.svg" alt="Angle"></a></li>
+                
+            </ul>
+
+            <a class="about-button-link button" href="#" rel="noopener noreferrer" target="_blank">Get my resume</a>
+        </div>
+
+        <div class="about-last-part">
+
+            <ul class="about-category">
+                <li class="about-li">
+                    <h3>Languages</h3>
+
+                    <nav class="about-icon">
+                        <img src="images/arrow-down.svg" alt="about_icon">
+                    </nav>
+                </li>
+            </ul>
+            <ul class="about-language">
+                <li class="about-lang-list">
+                    <img src="images/js.svg" alt="JavaScript">
+                    <p>JavaScript</p>
+                </li>
+                <li class="about-lang-list">
+                    <img src="images/html.svg" alt="HTML">
+                    <p>HTML</p>
+                </li>
+                <li class="about-lang-list">
+                    <img src="images/css.svg" alt="css">
+                    <p>CSS</p>
+                </li>
+            </ul>
+            <ul class="about-category">
+                <li class="about-li">
+                    <h3>Frameworks</h3>
+
+                    <nav class="about-icon">
+                        <img src="images/arrow-right.svg" alt="about_icon">
+                    </nav>
+                </li>
+                <li class="about-li">
+                    <h3>Skills</h3>
+
```

Comment:
- [ ] Kindly use the same color as the Figma design for your `h3` tag.

Reply:
Done: Changed the color to `color: #344563;`

Reply:
[OPTIONAL]: Please ensure that you replicate the design as much as possible (e.g., font, colors, images, tests, margins) using [the info from the Figma templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/161204178-db11e82a-8e00-4fcc-b060-6c5ed020691f.png)  | ![image](https://user-images.githubusercontent.com/10332499/161204121-d7608570-ab2a-4789-963c-8457e03ebe67.png)